src/analysis/retail/evoker/augmentation/modules/features/BuffTargetHelper/BuffTargetHelper.tsx

```diff
@@ -138,6 +160,24 @@ class BuffTargetHelper extends Analyzer {
       });
     });
   }
+
+  onEbonApply(event: ApplyBuffEvent) {
+    this.ebonApplyTimestamps.push(event.timestamp);
+  }
+
+  onEbonRefresh(event: RefreshBuffEvent) {
+    this.ebonRemoveTimestamps.push(event.timestamp);
+    this.ebonApplyTimestamps.push(event.timestamp);
+  }
+
+  onEbonRemove(event: RemoveBuffEvent) {
+    this.ebonRemoveTimestamps.push(event.timestamp);
+  }
+
+  onEbonRemoveFightEnd(event: FightEndEvent) {
+    this.ebonRemoveTimestamps.push(event.timestamp);
+  }
+
```

Comment:
You should check if you currently have EM active before pushing in a removal event.